bullet_train/app/views/showcase/engine/_stylesheets.html.erb

```diff
@@ -1 +1 @@
-<%= stylesheet_link_tag "application", "application.light", "showcase" %>
+<%= stylesheet_link_tag "application", "application.light", "showcase", "showcase.highlights" %>
```

Comment:
I'm gonna fix this up so client apps don't need to have control over including the showcase and showcase.highlights assets.